views/dapp/v2/components/layout/header/menu/profile/menu-switch-account/index.tsx

```diff
@@ -0,0 +1,117 @@
+import { Box, Button, Motion, Typography } from '@interest-protocol/ui-kit';
+import { formatAddress } from '@mysten/sui.js';
+import { useWalletKit } from '@mysten/wallet-kit';
+import { useTranslations } from 'next-intl';
+import { FC, useMemo } from 'react';
+import { toast } from 'react-hot-toast';
+
+import { CopySVG, UserSVG } from '@/components/svg/v2';
+import { wrapperVariants } from '@/constants';
+import { useWeb3 } from '@/hooks';
+import { ArrowLeft } from '@/svg';
+import { capitalize } from '@/utils';
+
+import MenuItemWrapper from '../../menu-item-wrapper';
+import { MenuSwitchAccountProps } from '../profile.types';
+
+const MenuSwitchAccount: FC<MenuSwitchAccountProps> = ({
+  isOpen,
+  loading,
+  suiNSRecord,
+  onBack,
+}) => {
+  const t = useTranslations();
+  const { account } = useWeb3();
+  const { accounts, selectAccount } = useWalletKit();
+
+  const copyToClipboard = (address: string) => {
+    window.navigator.clipboard.writeText(address || '');
+    toast(capitalize(t('common.v2.wallet.copy')));
+  };
+  const colorsArray = useMemo(
+    () =>
+      ['#fde68a', '#fed7aa', '#d9f99d', '#a5f3fc', '#e9d5ff', '#fecaca'].sort(
```

Comment:
These are the semantic colors we have. Use it with the "dark" and try globalizing them. We have other tasks we'll need it
  
  const COLORS = [
    dark ? '#BEF264' : '#84CC16',
    dark ? '#FCA5A5' : '#EF4444',
    dark ? '#FDBA74' : '#F97316',
    dark ? '#67E8F9' : '#06B6D4',
    dark ? '#FDBA74' : '#F59E0B',
    dark ? '#D8B4FE' : '#A855F7',
    dark ? '#F9A8D4' : '#EC4899',
  ];